run_web.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# wait for MYSQL server to start
+sleep 45
```

Comment:
Yes. I am looking into a program called "wait-for-it".sh and that could be the key